### ISSUE TITLE
NameError: name 'cca' is not defined

### ISSUE THREAD

**Issue by nboughton** [higlass/higlass-manage#24]
On Ubuntu 18.04, installed higlass-manage via pip3 and had to also install the following packages as build dependencies:

zlib1g-dev
libpng-dev
libssl-dev

When attempting to ingest a .bed file I get the following error:

```
higlass-manage ingest all_fDanKya3_1.sort.filteredPrimary.COVERAGE_ZERODEPTH_4gEVAL.bed --filetype bedfile --datatype bedlike --name fDanKya3_ZERODEPTH --project-name fDanKya3 --chromsizes-filename my.genome
Aggregating bedfile
Traceback (most recent call last):
  File "/usr/local/bin/higlass-manage", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/higlass_manage/ingest.py", line 45, in ingest
    project_name
  File "/usr/local/lib/python3.6/dist-packages/higlass_manage/ingest.py", line 67, in _ingest
    (to_import, filetype) = aggregate_file(filename, filetype, assembly, chromsizes_filename, has_header, no_upload, td)
  File "/usr/local/lib/python3.6/dist-packages/higlass_manage/ingest.py", line 83, in aggregate_file
    cca._bedfile(filename,
NameError: name 'cca' is not defined
```
I note that cca is not defined or imported in ingest.py.

**Comment by nboughton** [comment]
To add to this I updated the install and got a new error:

```
higlass-manage ingest all_fDanKya3_1.sort.filteredPrimary.COVERAGE_ZERODEPTH_4gEVAL.bed --filetype bedfile --datatype bedlike --name fDanKya3_ZERODEPTH --project-name fDanKya3 --chromsizes-filename my.genome
Traceback (most recent call last):
  File "/root/hg-env/env/bin/higlass-manage", line 11, in <module>
    sys.exit(cli())
  File "/root/hg-env/env/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/root/hg-env/env/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/root/hg-env/env/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/hg-env/env/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/root/hg-env/env/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/root/hg-env/env/lib/python3.6/site-packages/higlass_manage/ingest.py", line 45, in ingest
    project_name
  File "/root/hg-env/env/lib/python3.6/site-packages/higlass_manage/ingest.py", line 61, in _ingest
    print('File not found:', filename, file=sys.stderr)
NameError: name 'sys' is not defined
```

I fixed this by adding "import sys" to the top of:
 
```
/root/hg-env/env/lib/python3.6/site-packages/higlass_manage/ingest.py
```

**Comment by pkerpedjiev** [comment]
Sorry about that. This should be fixed in `v0.6.1`. Please reopen this issue if that's not the case.
